content/1.7/back-office/catalog/stock/stock.md

```diff
@@ -0,0 +1,31 @@
+
+---
+title: Stock
+weight: 1
+---
+
+  
+# SPECS Stock & Stock movement
+
+## Stock
+
+  
+| Stock's type | Definition | When impacted | 
+|-----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Physical stock | Number of products that are physically in my stock.| When a merchant can add or remove physical stock 
```

Comment:
```suggestion
| Physical stock | Number of products that are physically in my stock.| When a merchant can add or remove physical stock, when an order is shipped |
```

Reply:
Not really, it is a merchant edit the physical stock so when it is shipped and when he increases or decrease it from the product page, stock page or from another source